tasks/masters-os-update-and-upgrade.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I'd like to rename this file to show that it is only called from os-update-and-upgrade. Maybe by creating a folder. I suggest `tasks/os-update-and-upgrade/masters.yml`

Reply:
Made change.